config/vufind/searchbox.ini

```diff
@@ -69,7 +69,7 @@ group[] = "Other Sites"
 ;layouts[] = "burmese"
 ;layouts[] = "chinese"
 ;layouts[] = "czech"
-;layouts[] = "english"
+layouts[] = "english"
```

Comment:
Is the english keyboard supposed to be enabled by default?

Reply:
This was a dev change that slipped through. Thank you!